CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,66 @@
 
 import Foundation
 
-print("Hello, World!")
+class Person {
+    var name: String
+    var age: UInt
+    var money: Int
+    
+    init?(name: String, age: UInt, money: Int) {
+        if (0...100).contains(age) == false || name.isEmpty {
+            return nil
+        }
+        self.name = name
+        self.age = age
+        self.money = money
+    }
+    
+    func order(_ coffee: CoffeeShop.Coffee, of coffeeShop: CoffeeShop, by name: String) {
+        if self.money < coffee.rawValue {
+            var insufficientFunds = coffee.rawValue - self.money
+            print ("잔액이 \(insufficientFunds)만큼 부족합니다.")
+        } else {
+            coffeeShop.make(_ : coffee, from: name)
+            self.money = money - coffee.rawValue
+        }
+    }
+}
+
+class CoffeeShop {
+    
+    var name: String
+    var barista: Person!
```

Comment:
해당 문법을 사용하는 경우는 아래와 같다고 생각합니다.
>Sometimes it’s clear from a program’s structure that an optional will always have a value, after that value is first set. In these cases, it’s useful to remove the need to check and unwrap the optional’s value every time it’s accessed, because it can be safely assumed to have a value all of the time.
UDI님이 말씀해주신 이유가 틀린 것은 절대 아니고 추가적으로 이해하고 계시면 좋을 것 같습니다!

Reply:
확인했습니다!